nativelib/src/main/scala/scala/scalanative/runtime/ieee754tostring/ryu/RyuDouble.scala

```diff
@@ -745,26 +739,49 @@ object RyuDouble {
    *    new offset as: old offset + number of created chars (i.e. last modified
    *    index + 1)
    */
-  @noinline def doubleToChars(
+  @noinline
```

Comment:
I think this `@noinline` adds no value. You may want to remove it now that the code has been restructured.  

The `@noinline` in the long routine remains relevant although I can not imagine a compiler which would inline that much code.

Reply:
Done in https://github.com/scala-native/scala-native/pull/2917/commits/3c79f9080e1018e4ae5ddea35494567767049ed4